src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let total = 0;
+
+for (const population of populations) {
```

Comment:
Consider using `forEach` or `reduce` instead of a traditional for loop for iterating over the populations array.